recipes-extended/shadow/shadow_%.bbappend

```diff
@@ -0,0 +1,12 @@
+FILESEXTRAPATHS:prepend:sota := "${THISDIR}/files:"
+
+SRC_URI:append:sota = " file://shadow.conf"
+
+do_install:append:sota() {
+    if ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'true', 'false', d)}; then
+        install -d ${D}${sysconfdir}/tmpfiles.d
+        install -m 0644 ${UNPACKDIR}/shadow.conf ${D}${sysconfdir}/tmpfiles.d/shadow.conf
```

Comment:
We should install at `${D}${nonarch_libdir}/tmpfiles.d ` instead, no need to use /etc for this file, as it is the system default behavior.